app/components/moderation-list/component.js

```diff
@@ -0,0 +1,76 @@
+import Ember from 'ember';
+/**
+ * Provides a list of pending, accepted, and rejected preprints. Provides filtering by preprint state and sorting based
+ * on the preprint creation date. If number of records in each type exceeds 10, pagination is enabled. Current page for
+ * each state is tracked and used by the paginator using the added pager service . Pagination is using the
+ * pagination-pager addon (source: https://github.com/knownasilya/pagination-pager).
+ *
+ * Sample usage:
+ * ```handlebars
+ * {{moderation-list
+ *    model=model
+ * }}
+ * ```
+ * @class moderation-list
+ **/
+export default Ember.Component.extend({
+    store: Ember.inject.service(),
+    params: Ember.inject.service('pager'),
+    theme: Ember.inject.service(),
+    screenWidth: 0,
+    loading: false,
+    sortOptions: Ember.computed( function() { // Sort options for moderation list
```

Comment:
Forget to add a Todo, but planning to pass i18n.locale to it when translation is ready